Classes/Hooks/TranslateHook.php

```diff
@@ -113,6 +113,24 @@ public function processTranslateTo_copyAction(string &$content, array $languageR
                         $targetLanguage = $site->getLanguageById($languageRecord['uid']);
                         $targetLanguageIso = $targetLanguage->getTwoLetterIsoCode();
 
+                        //Use hreflang for RFC 1766 / 3066 standard lang code
+                        if (method_exists($targetLanguage, 'getHreflang')) {
+                            $targetHreflang = $targetLanguage->getHreflang();
+                            if ($targetHreflang != '') {
+                                $targetLanguageIso = $targetHreflang;
+                            }
+                        } else {
+                            $targetLanguageIso = $targetLanguage->getTwoLetterIsoCode();
+                        }
+
+                        //check for translation target language defined explicitly through extension configuration
+                        if (isset($GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']['wv_deepltranslate']['translationTargetLanguage'])) {
+                            $translationTargetLanguage = $GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']['wv_deepltranslate']['translationTargetLanguage'];
+                            if ($translationTargetLanguage != '') {
+                                $targetLanguageIso = $translationTargetLanguage;
+                            }
```

Comment:
As I see, there is no check, if the given specific string only contains the ISO-639-1 language key or if there is more input in defined string. This should be checked, before giving the input to variable.